TTCN3_Project/javasrc/CANPortFilter/CANPortFilterProvider.java

```diff
@@ -0,0 +1,14 @@
+package CANPortFilter;
+
+import com.testingtech.ttcn.tri.PortFilter;
+import com.testingtech.ttcn.tri.extension.PortFilterProvider;
+
+public class CANPortFilterProvider implements PortFilterProvider {
+
+	@Override
+	public PortFilter getPortFilter() {
+		// TODO Auto-generated method stub
+		return new CANPortFilter();
```

Comment:
This whole class does nothing except returning an object of type `CANPportFilter()`. So: Why is it necessary?